frontend/packages/dev-console/src/components/add/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -57,7 +57,7 @@ export const DeveloperFeaturesGettingStartedCard: React.FC = () => {
       id="developer-features"
       icon={<FlagIcon color="var(--pf-global--palette--orange-300)" aria-hidden="true" />}
       title={t('devconsole~Explore new developer features')}
-      titleColor={'var(--pf-global--palette--gold-700)'}
+      titleColor={'var(--co-global--palette--gold-700)'}
```

Comment:
It stands out and looks somewhat strange that originally someone used both `gold` and `orange`, compared to the other icon & text heading being either `purple` or `blue`.

Personally I'd switch the text to `orange` so it matches the flag icon color theme. And use a dark `orange` on the light theme. Like so...
```suggestion
      titleColor={'var(--co-global--palette--orange-700)'}
```
With switching from `gold` to `orange` vars
<img width="1174" alt="Screen Shot 2022-03-09 at 1 45 19 PM" src="https://user-images.githubusercontent.com/1874151/157517126-17c04795-f038-4665-9550-77377fe040f2.png">
**Light**
<img width="886" alt="Screen Shot 2022-03-09 at 2 24 24 PM" src="https://user-images.githubusercontent.com/1874151/157517178-83feb622-abce-4a3e-8c83-8a699c03b0b7.png">